project_2.html

```diff
@@ -0,0 +1,603 @@
+							<td>3</td>
+						</tr>
+						<tr>
+							<td>
+								<a class="pj-part"
+									data-bs-content="<img src='assets/images/parts/ops_parts_8pinDipSocket.png' height='150px' width='auto'>"
+									data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0">
+									Dip Socket, 8 pin
+								</a>
+							</td>
+							<td>1</td>
+						</tr>
+						<tr>
+							<td>
+								<a class="pj-part"
+									data-bs-content="<img src='assets/images/parts/ops_part_led.jpg' height='150px' width='auto'>"
+									data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0" 
+									href="./assets/datasheets/ops_all_leds_datasheet.pdf" target="_blank" rel="noopener noreferrer">
+									LED, 3.2V
+								</a>
+							</td>
+							<td>1</td>
+						</tr>
+						<tr>
+							<td>Capacitor, 0.1µF</td>
+							<td>1</td>
+						</tr>
+						<tr>
+							<td>Capacitor, 1µF</td>
+							<td>1</td>
+						</tr>
+						<tr>
+							<td>Capacitor, 10µF</td>
+							<td>1</td>
+						</tr>
+						<tr>
+							<td> 
+								<a class="pj-part"
+									data-bs-content="<img src='assets/images/visuals/ops_proj2_pcb.png' height='150px' width='auto'>"
+									data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0">
+									555 Piano PCB
+								</a>
+							</td>
+							<td>1</td>
+						</tr>
+						<tr>
+							<td> 
+								<a class="pj-part"
+									data-bs-content="<img src='assets/images/parts/ops_part_buzzer.webp' height='150px' width='auto'>"
+									data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0" a class="link-primary" 
+									href="https://drive.google.com/file/d/10nJyWBjgadDRzMUlxaWnqGHkBlo_iqCw/view?usp=sharing" target="_blank" rel="noopener noreferrer">
+									Piezo Buzzer 
+								</a>
+							</td>
+							<td>1</td>
+						</tr>
+						<tr>
+							<td>Jumper Wire</td>
+							<td>?</td>
+						</tr>
+					</tbody>
+				</table>
+				<!-- End Part Table -->
+
+<h2 class="mb-3 fw-bold" id="pj-heading-schematics">Schematics</h2>
+			<div class="accordion mb-5" id="schemAccordian">
+				<div class="accordion-item">
+					<h2 class="accordion-header">
+						<button class="accordion-button" type="button" data-bs-toggle="collapse"
+							data-bs-target="#schemCollapse1" aria-expanded="true" aria-controls="schemCollapse1">
+							Schematic A (555 LED Blinker)
+						</button>
+					</h2>
+					<!--Pin to page?-->
+					<div id="schemCollapse1" class="accordion-collapse collapse show" data-bs-parent="#schemAccordian">
+						<div class="accordion-body text-center">
+							<img src='assets/images/visuals/ops_schem_proj2a.png' class="pj-schem">
+						</div>
+					</div>
+				</div>
+
+				<div class="accordion-item">
+					<h2 class="accordion-header">
+						<button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
+							data-bs-target="#schemCollapse2" aria-expanded="true" aria-controls="schemCollapse2">
+							Schematic B (555 Piano)

```

Comment:
![image](https://github.com/uci-ieee/ops-webpages-2023-2024/assets/45702229/6b455fb4-9076-495a-b3f0-fc06a83b4fea)
![image](https://github.com/uci-ieee/ops-webpages-2023-2024/assets/45702229/5f0642ae-40ae-45c1-a089-210254a1bb35)

Please update the Schematic B to be a more legible sizing (consistent with Schematic A)